src/test/java/org/opentripplanner/transit/model/site/GroupStopTest.java

```diff
@@ -27,6 +32,57 @@ class GroupStopTest {
     .addLocation(STOP_LOCATION)
     .build();
 
+  @Test
+  void testGroupStopGeometry() {
+    StopLocation stopLocation1 = TEST_MODEL.stop("1:stop", 1d, 1d).build();
+    StopLocation stopLocation2 = TEST_MODEL.stop("2:stop", 2d, 2d).build();
+
+    GroupStop groupStop = StopModel
+      .of()
+      .groupStop(TransitModelForTest.id(ID))
+      .withName(NAME)
+      .addLocation(stopLocation1)
+      .addLocation(stopLocation2)
+      .build();
+
+    Geometry groupStopGeometry1 = Objects.requireNonNull(groupStop.getGeometry()).getGeometryN(0);
+    assertEquals(stopLocation1.getGeometry(), groupStopGeometry1);
+
+    Geometry groupStopGeometry2 = Objects.requireNonNull(groupStop.getGeometry()).getGeometryN(1);
+    assertEquals(stopLocation2.getGeometry(), groupStopGeometry2);
+  }
+
+  @Test
+  void testGroupStopEncompassingAreaGeometry() {
+    Geometry encompassingAreaGeometry = GeometryUtils
+      .getGeometryFactory()
+      .toGeometry(new Envelope(1d, 2d, 1d, 2d));
```

Comment:
This is totally fine but we have convenience constants for Geometries that you can use next time:

- https://github.com/opentripplanner/OpenTripPlanner/blob/9e258f5a4637feb4a528c2e37e515e60e2a6e98e/src/test/java/org/opentripplanner/_support/geometry/Coordinates.java
- https://github.com/opentripplanner/OpenTripPlanner/blob/028efcc5db3e21928ae79f0ce689407d75f96150/src/test/java/org/opentripplanner/_support/geometry/Polygons.java

Reply:
Nice! These are useful. Made the change.